circuits/license/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+### Changed
+
+## [0.1.0] - 2023-05-02
```

Comment:
You miss the version hyperlink (look at other CHANGELOG)

Reply:
> You miss the version hyperlink (look at other CHANGELOG)

I see hyperlink is there and it works, just like in the CHANGELOG for transfer, I am not sure what to do

Reply:
My bad, it was about the `Unreleased` hyperlink